python/lsst/daf/butler/core/config.py

```diff
@@ -58,6 +59,24 @@
     yamlLoader = yaml.SafeLoader  # type: ignore
 
 
+def _uuid_representer(dumper, data):
+    """Generate YAML representation for UUID.
+
+    This produces a scalar node with a tag "!uuid" and value being a regular
+    string representation of UUID.
+    """
+    return dumper.represent_scalar("!uuid", str(data))
+
+
+def _uuid_constructor(loader, node):
+    if node.value is not None:
+        return uuid.UUID(hex=node.value)
+
+
+yaml.Dumper.add_representer(uuid.UUID, _uuid_representer)
```

Comment:
This is for import/export, somehow I though it would be nice to have it defined globally, but I can also move it to yaml import/export backend.

Reply:
Both classes are going to be loaded on code import anyhow so it doesn't really make a difference. As written it implies that UUID are supported in our configs so that was why I had to think a bit. Moving it makes it more explicit.